test/hotspot/jtreg/compiler/valhalla/inlinetypes/TestNullableArrays.java

```diff
@@ -2556,13 +2563,14 @@ public void test96_verifier(boolean warmup) {
     }
 
     // Test loads from vararg arrays
-    @Test(failOn = LOAD_UNKNOWN_INLINE)
+    @Test
+    @IR(failOn = {LOAD_UNKNOWN_INLINE})
```

Comment:
Can be simplified to `failOn = LOAD_UNKNOWN_INLINE`.

Reply:
see previous comment